suite.lisp

```diff
@@ -0,0 +1,77 @@
+(in-package #:org.shirakumo.trivial-benchmark)
+
+(defvar *benchmark-packages* (make-hash-table :test 'eq)
+  "A table mapping package objects (which are benchmark packages) to a list of fbound symbols.")
+
+(defun find-benchmark-package (package-designator)
+  "Find the benchmark package designated by PACKAGE-DESIGNATOR and return it, otherwise return NIL if it doesn't exist."
+  (let ((package-object (find-package package-designator)))
+    (if (or (null package-object)
+            (null (nth-value 1 (gethash package-object *benchmark-packages*))))
+        nil
+        package-object)))
```

Comment:
I try not to abuse the conflation of values with booleans.

`(null x)` checks if `x` is the object `nil`. To me, this is not semantically equivalent to "is `x`  false`.

What I should do is change the second `null` to a `not` however.